usaspending_api/references/management/commands/loadagencies.py

```diff
@@ -16,33 +16,44 @@ def handle(self, *args, **options):
 
         try:
             with open(os.path.join(django.conf.settings.BASE_DIR,
-                      'usaspending_api/references/management/commands/agency_list.csv')) \
-                      as csvfile:
+                      'usaspending_api', 'data', 'authoritative_agency_list.csv'), encoding="Latin-1") \
+                    as csvfile:
 
                 reader = csv.DictReader(csvfile)
                 for row in reader:
```

Comment:
If a human will be periodically providing updates to the agency .csv file, we might want to think about explicitly lower or upper casing the incoming column headers so the loader works if someone changes the case on a future version of the file. People sometimes don't realize that changing the headings impacts the load.